README.md

```diff
@@ -37,6 +37,38 @@ General help with the software and the website is also available on the IRC
 channel at [irc.esper.net#cytube](http://webchat.esper.net/?channels=cytube)
 during US daytime hours.
 
+Docker
+------
+
+Example for using the dockerfile on this repo.
+
+```
+docker build -t sync .
+docker network create sync
+
+docker run -d --name sync-db \
+-e MARIADB_ROOT_PASSWORD='abcdefg123456' \
+-e MARIADB_DATABASE=cytube \
+-e MARIADB_USER=cytube \
+-e MARIADB_PASSWORD=aaaaa \
+--network sync mariadb
+
+docker run -d --name sync-web \
+-e MYSQL_HOST=sync-db \
+-e MYSQL_PASSWORD=aaaaa \
+-e ROOT_URL=https://cytube.my.domain \
+-e IO_ROOT_URL=https://cytube.my.domain \
+-e ROOT_DOMAIN=cytube.my.domain \
+-e VIRTUAL_HOST=cytube.my.domain \
+-e VIRTUAL_PORT=8080 \
+-e LETSENCRYPT_HOST=cytube.my.domain \
+-e YOUTUBE_KEY=abcdefg \
+--network sync sync
+```
+
+Feedback
```

Comment:
What's this blank section for?

Reply:
Its just a space between the contents and the feedback section.

Reply:
Sorry, I meant the Feedback section itself is blank (there is a heading but nothing under it).